src/site/content/en/lighthouse-performance/resource-summary/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Keep request counts low and transfer sizes small
+description: |
+  An overview of how high resource counts and large transfer sizes affect load performance, 
+  and strategies for reducing request counts and transfer sizes.
+web_lighthouse:
+  - resource-summary
+---
+
+The **Keep request counts low and transfer sizes small** audit tells you how many network requests 
+were made and how much data was transferred while your page loaded.
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="resource-summary.jpg" 
+       alt="Keep request counts low and transfer sizes small">
+  <figcaption class="w-figcaption">
+    <b>Keep request counts low and transfer sizes small</b>
+  </figcaption>
+</figure>
+
```

Comment:
I turned the aside into an unordered list

Reply:
This was much easier for me to follow now that it's chunked! Thanks for doing this!